nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/.github/pull_request.md

```diff
@@ -0,0 +1,8 @@
+**Before submitting a pull request,** please make sure the following is done:
+
+1. Fork the [nf-core/{{ cookiecutter.pipeline_slug }} repository](https://github.com/nf-core/{{ cookiecutter.pipeline_slug }}) and create your branch from `master`.
+2. If you've fixed a bug or added code that should be tested, add tests!
+3. Ensure the test suite passes (`nextflow run . -profile test,docker`).
+4. Make sure your code lints (`nf-core lint`).
```

Comment:
For consistency with the above command, should be `nf-core lint .`

May be clearer to use `<pipeline dir>` instead of `.`?